tests/test_xbue_calendar.py

```diff
@@ -0,0 +1,132 @@
+from unittest import TestCase
+import pandas as pd
+from pytz import UTC
+
+from .test_trading_calendar import NoDSTExchangeCalendarTestBase
+from trading_calendars.exchange_calendar_xbue import XBUEExchangeCalendar
+
+
+class XBUECalendarTestCase(NoDSTExchangeCalendarTestBase, TestCase):
```

Comment:
From light googling I see that Argentina used to observe DST before 2009. What do you think of enabling the DST tests and overriding the dates to something pre-2009?

Reply:
Looks like it was observed from 2007-2009.  I enabled the tests and added 2008 daylight savings dates